.github/workflows/release-publish.yml

```
@@ -161,3 +161,45 @@ jobs:
         file: utils/automation/Dockerfile.ci
         labels: |
           org.opencontainers.image.version=${{ steps.get_release.outputs.tag_name }}.0
+
+  nomossa-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+          build-essential pkg-config \
+          libglib2.0-dev librpm-dev \
+          libxml2-dev libxslt1-dev \
+          libjson-c-dev
+
+      - name: Setup CMake
+        uses: lukka/get-cmake@v4.0.3
+
+      - name: Build Nomossa
+        run: |
+          mkdir -p nomossa-build
+          cmake ./src/nomos -DCMAKE_BUILD_TYPE=MinSizeRel -B nomossa-build
```

```suggestion
         cmake -DCMAKE_BUILD_TYPE=MinSizeRel -DAGENTS="nomos" -DBUILD_SHARED_LIBS="off" -DCMAKE_EXE_LINKER_FLAGS="-static" -B nomossa-build -S . -G Ninja
```

We need to compile this statically and link the libraries. Use the flag I have provided in the command to do so.